src/components/AccessibilityPolicyForm/index.jsx

```diff
@@ -0,0 +1,278 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Button from '@edx/paragon/src/Button';
+import InputText from '@edx/paragon/src/InputText';
+import StatusAlert from '@edx/paragon/src/StatusAlert';
+import TextArea from '@edx/paragon/src/TextArea';
+
+import { clearAccessibilityStatus, submitAccessibilityForm } from '../../data/actions/accessibility';
+import { accessibilityActions } from '../../data/constants/actionTypes';
+import styles from './AccessibilityPolicyForm.scss';
+
+export class AccessibilityPolicyForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      formEmail: '',
+      formFullName: '',
+      formMessage: '',
+      statusAlertOpen: false,
+      validFormContent: true,
+      validationMessages: [],
+    };
+
+    this.statusAlertRef = {};
+    this.emailInputRef = {};
+    this.fullNameInputRef = {};
+    this.messageInputRef = {};
+    this.submitButtonRef = {};
+
+    this.handleChange = this.handleChange.bind(this);
+    this.validateFormContent = this.validateFormContent.bind(this);
+    this.closeStatusAlert = this.closeStatusAlert.bind(this);
+    this.renderStatusAlert = this.renderStatusAlert.bind(this);
+  }
+
+  onSubmitClick() {
+    this.props.clearAccessibilityStatus();
+    this.setState({ // this will clear on every submit (success/fail) - correct?
+      formEmail: '',
+      formFullName: '',
+      formMessage: '',
+    });
+
+    const { formEmail, formFullName, formMessage } = this.state;
+    const validateEmail = this.validateFormContent('email', formEmail);
```

Comment:
Why have one validation method? Why not have a `validateEmailEmail` and a `validateName` and a `validateMessage` method? You're essentially doing this by`if...else-if`ing at the top level of `validateFormContent` - why not move that logic into their own functions? The benefits I see are that it makes testing easier as you're testing smaller modules or logic, and it (slightly) increases readability by taking a relatively generic name like `validateFormContent` and making it more specific (`validateEmailAddress`).